app/controllers/petitions_controller.rb

```diff
@@ -12,7 +12,15 @@ def index
   end
 
   def show
-    respond_with @petition = Petition.visible.find(params[:id])
+    begin
```

Comment:
Yes, you could. I prefer explicit blocks because they fold nicer in my text editor - it's purely a subjective decision.